sel/src/Sel/Hashing/Password.hs

```diff
@@ -182,6 +200,18 @@ passwordHashToHexText = Base16.encodeBase16 . passwordHashToBinary
 -- @since 0.0.1.0
 newtype Salt = Salt StrictByteString
 
+-- | @since 0.0.1.0
+instance Eq Salt where
+  (Salt s1) == (Salt s2) = s1 == s2
+
+-- | @since 0.0.1.0
+instance Ord Salt where
+  (Salt s1) `compare` (Salt s2) = s1 `compare` s2
+
+-- | @since 0.0.1.0
+instance Show Salt where
+  show (Salt s1) = show s1
```

Comment:
I believe there's been a misunderstanding, "explicit strategies" refers to this: https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/deriving_strategies.html

Reply:
Ah. Sorry about that.

Reply:
no worries. :)